_includes/form-join.html

```diff
@@ -55,6 +55,12 @@
         />
       </div>
 
+      <div>
+        <div class="control">
+          <label for="receive_updates">{% t connect.receive_updates %}</label>
```

Comment:
Non-blocking: I feel they should be inline, have a break between the two doesn't really fit UX patterns. 

Likewise, I feel the language could be less passive? "Sign up to receive occasional updates"

Reply:
I totally agree, I was aiming for consistency within the form but agree it is better to align with broader patterns. Updated wording and styling in: ab03af5b070f5eaafac0f5acbb0d52cd8fc035fd